birdhouse/components/cowbird/default.env

```diff
@@ -4,9 +4,9 @@
 
 # All env in this default.env can be overridden by env.local.
 
-# All env in this default.env must NOT depend on any other env.  If they do,
-# must use single quote to avoid early expansion before overrides in local.env
-# are applied and must add to the list of DELAYED_EVAL.
+# All env in this default.env must NOT depend on any other env.  If they do, they
+# must use single quotes to avoid early expansion before overrides in env.local
```

Comment:
Woops, `local.env`, not sure what I was smoking at that moment, thanks for all the fixes.

Reply:
I did the same mistake multiple times.